lib/statemanager.js

```diff
@@ -594,7 +594,7 @@ StateManager.prototype.processGasEstimate = function (from, rawTx, blockNumber,
       }
       var result = '0x0'
       if (!results.error) {
-        result = to.hex(results.gasUsed)
+        result = results.gasRefund ? to.hex(results.gasUsed.add(results.gasRefund)) : to.hex(results.gasUsed);
```

Comment:
We expect the `gasRefund` field to always be there, right? If so, this squelches an error and makes things look like they're working if the `gasRefund` field suddenly disappears.

Reply:
fixed in latest commits